integration/benchmark/cmd/ci/main.go

```diff
@@ -59,8 +59,8 @@ const (
 	serviceAccountPrivateKeyHex = unittest.ServiceAccountPrivateKeyHex
 
 	// Auto TPS scaling constants
-	additiveIncrease       = 50
-	multiplicativeDecrease = 0.8
+	additiveIncrease       = 100
+	multiplicativeDecrease = 0.9
```

Comment:
@pattyshack you were right!  0.9 looks way more "gentle" on the graphs!

Reply:
it's well known that tcp backoff factor is too aggressive.